hw/ip/csrng/doc/_index.md

```diff
@@ -16,21 +16,23 @@ In this document the terms "DRNG" and "TRNG" are used most generally to refer to
 However, the terms "DRBG" or "NRBG" are specifically used when respectively referring to SP 800-90A or SP 800-90C requirements.
 Meanwhile, when addressing requirements which originate from AIS31 we refer to the specific DRG.3 or PTG.3 classes of RNGs.
 
-This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [Comportable guideline for peripheral functionality]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure and software-inaccessible transmission of random numbers. The bus connections to peripheral modules is done using the CSRNG application interface. This interface allows peripherals to manage CSRNG instances, and request for obfuscated entropy to be returned from the CSRNG module.
+This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [comportability definition and specification]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure and software-inaccessible transmission of random numbers.
+The bus connections to peripheral modules is done using the CSRNG application interface.
```

Comment:
"is done" -> "are done" (or change "connections" -> "connection")

Reply:
Good catch!